server/src/main/java/org/opensearch/cluster/service/MasterThrottlingStats.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.cluster.service;
+
+import org.opensearch.common.metrics.CounterMetric;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Contains stats of Master Task Throttling.
+ * It stores the total cumulative count of throttled tasks per task type.
+ */
+public class MasterThrottlingStats {
```

Comment:
A useful stat would be how long since a task was throttled, but not yet released?  Would be helpful in debugging- basically oldest throttled task. What do you think?